syntax = "proto3";

package hello;

service SuggestionsService {
    rpc SuggestionsLogic (SuggestionsRequest) returns (SuggestionsResponse);
}

message SuggestionsRequest {
}

message SuggestionsResponse {
    string book1id = 1;
    string book1name = 2;
    string book1author = 3;
    string book2id = 4;
    string book2name = 5;
    string book2author = 6;
}